src/main/java/com/klarna/hiverunner/StandaloneHiveServerContext.java

```diff
@@ -158,10 +156,6 @@ protected void configureAssertionStatus(HiveConf conf) {
                 false);
     }
 
-    protected void configureCheckForDefaultDb(HiveConf conf) {
```

Comment:
This one was old legacy even in HiveRunner 2.6.0. Since we use an in-mem DB these days this is not needed.

The reason I removed it was that Hive now seems to check if someone tries to set an unused/deprecated hive property and I wanted to remove as much 'noise' as possible to be able to see if there were any other important issues.